Subject: Use GCS Maven Central mirror for buildscript dependencies
Origin: DHI build reliability
Reference: https://maven-central.storage-download.googleapis.com/maven2/
Description: spark-kubernetes-operator already mirrors Maven Central for
  subproject repositories, but buildscript resolves plugins.gradle.org
  classpath artifacts that redirect to repo.maven.apache.org and receive 429
  responses from DHI cloud builders. Add the same GCS mirror to buildscript
  repositories before the plugin portal.

diff --git a/build.gradle b/build.gradle
--- a/build.gradle
+++ b/build.gradle
@@ -19,6 +19,10 @@
  */
 buildscript {
   repositories {
+    maven {
+      name = "GCS Maven Central mirror"
+      url = uri("https://maven-central.storage-download.googleapis.com/maven2/")
+    }
     maven {
       url = uri("https://plugins.gradle.org/m2/")
     }
